packages/dds/tree/src/test/simple-tree/typesUnsafe.spec.ts

```diff
@@ -0,0 +1,14 @@
+/*!
+ * Copyright (c) Microsoft Corporation and contributors. All rights reserved.
+ * Licensed under the MIT License.
+ */
+
+// eslint-disable-next-line import/no-internal-modules
+import type { ReadonlyMapInlined } from "../../simple-tree/typesUnsafe.js";
+// eslint-disable-next-line import/no-internal-modules
+import type { numberSchema } from "../../simple-tree/leafNodeSchema.js";
+import type { areSafelyAssignable, requireTrue } from "../../util/index.js";
+
+type MapInlined = ReadonlyMapInlined<string, typeof numberSchema>;
```

Comment:
Nit: `ReadonlyMapInlined` is defined in `internalTypes` - should this test be in `internalTypes.spec.ts`?

Reply:
the code this is testing is in the file this is the test for (typesUnsafe.ts)

internalTypes just has a reexport for it, and this is not testing the reexport, its testing the definition.